src/components/Graph.react.js

```diff
@@ -474,6 +474,11 @@ const graphPropTypes = {
             'reset+autosize',
         ]),
 
+	/**
+	 * Delay for registering a double-click event in ms
+	 */
+	doubleClickDelay: PropTypes.number,
```

Comment:
@Marc-Andre-Rivet The prop in plotly.js [has min/max values defined](https://github.com/pynklu/plotly.js/blob/9dcbc716c81ed995020f31a2622e73cf67bccb46/src/plot_api/plot_config.js#L178), but I don't think there's a way to do that here. Should I add the min/max values to the docstring?

Reply:
You're right. Proptypes support runtime validation but our use of react-docgen does not. The docstring is best. :)